### PR TITLE
feat: add Download, Edit, History, Plus, Pause, & Trash icons

### PR DESCRIPTION
**Pull request by joel-jaimon**
This PR includes all the required icons for the new merchant reporting ui ([figma link](https://www.figma.com/file/5t4XMQ0PQHAvvcls614c6o/Merchant-Reporting?node-id=435%3A11894)).

**The following icons are added:** 
1. Pause Icon
2. Trash Icon
3. History Icon
4. Plus Icon
5. Download Icon
6. Edit Icon

Screenshot 2022-07-01 at 12 05 11 AM

### CODE REVIEW

**Review by chaitanyadeorukhkar — CHANGES_REQUESTED** [review]
Some of the svgs are exported incorrectly from figma. Since we don't have a guideline in place on how to do this, we can get on a call and I'll try to help you. We'll add a proper guideline soon.

**Comment by joel-jaimon** [comment]
> Some of the svgs are exported incorrectly from figma. Since we don't have a guideline in place on how to do this, we can get on a call and I'll try to help you. We'll add a proper guideline soon.

@chaitanyadeorukhkar As discussed in the call, I've made the required changes. Please check.

File: packages/blade/src/components/Icons/iconMap.ts [hunk 1]
```diff
@@ -8,6 +8,13 @@ import DollarIconComponent from './DollarIcon';
 import EyeIconComponent from './EyeIcon';
 import EyeOffIconComponent from './EyeOffIcon';
 import RupeeIconComponent from './RupeeIcon';
+import TrashIconComponent from './TrashIcon';
+import PauseIconComponent from './PauseIcon';
+import EditIconComponent from './EditIcon';
+import DownloadIconComponent from './DownloadIcon';
+import PlusIconComponent from './PlusIcon';
+import HistoryIconComponent from './HistoryIcon';
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/Icons/iconMap.ts:16]: Can you sort all the icons alphabetically?

File: packages/blade/src/components/Icons/iconMap.ts [hunk 2]
```diff
@@ -22,6 +29,12 @@ const iconMap: Record<string, React.ComponentType<IconProps>> = {
   EyeIcon: EyeIconComponent,
   EyeOffIcon: EyeOffIconComponent,
   RupeeIcon: RupeeIconComponent,
+  TrashIcon: TrashIconComponent,
+  PauseIcon: PauseIconComponent,
+  EditIcon: EditIconComponent,
+  DownloadIcon: DownloadIconComponent,
+  PlusIcon: PlusIconComponent,
+  HistoryIcon: HistoryIconComponent,
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/Icons/iconMap.ts:37]: Sort?

File: packages/blade/src/components/Icons/index.tsx [hunk 3]
```diff
@@ -30,3 +30,9 @@ export { default as DollarIcon } from './DollarIcon';
 export { default as EyeIcon } from './EyeIcon';
 export { default as EyeOffIcon } from './EyeOffIcon';
 export { default as RupeeIcon } from './RupeeIcon';
+export { default as TrashIcon } from './TrashIcon';
+export { default as PauseIcon } from './PauseIcon';
+export { default as EditIcon } from './EditIcon';
+export { default as DownloadIcon } from './DownloadIcon';
+export { default as PlusIcon } from './PlusIcon';
+export { default as HistoryIcon } from './HistoryIcon';
```
Comment (chaitanyadeorukhkar) [inline packages/blade/src/components/Icons/index.tsx:38]: Sort? 😅

**Review by chaitanyadeorukhkar — CHANGES_REQUESTED** [review]
Can you take a merge from master & resolve conflicts? Just need to sort exports otherwise LGTM

**Pull request merged**